README.md

```diff
@@ -205,3 +206,17 @@ The tree and file metadata are repopulated into memory on an interval (currently
 ### Auth
 
 Authentication with the Google Drive v3 api is handled by the auth.js file, which exposes a single method `getAuth`. `getAuth` will either return an already instantiated authentication client or produce a fresh one. Calling `getAuth` multiple times will not produce a new authentication client if the credentials have expired; we should build this into the auth.js file later to automatically refresh the credentials on some sort of interval to prevent them from expiring.
+
+### User Authentication
+
+Library currently supports both Slack and Google Oauth methods. As Library sites are intended to be internal to your organization, Oauth with your organization is strongly encouraged. To use Slack Oauth, specify your Oauth strategy in your `.env` file, like so:
```

Comment:
```suggestion
Library currently supports both Slack and Google Oauth methods. As Library sites are usually intended to be internal to a set of limited users, Oauth with your organization is strongly encouraged. To use Slack Oauth, specify your Oauth strategy in your `.env` file, like so:
```
We've heard from the community that a pattern for running a Library site without any auth at all (for [publi use cases](#156)) would be nice. Maybe we could keep this language open to that possibility in the future?

In a future PR, we might consider adding additional OAUTH_STRATEGY values (like `public` or `disabled`) that would hide some of the user chrome and noop the oauth middleware to open up Library sites to the public.